### PR TITLE
Bump core foundation

### PR DESCRIPTION
**Pull request by glennw**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "offscreen_gl_context"
 license = "MIT / Apache-2.0"
-version = "0.11.1"
+version = "0.12.0"
```
Comment (mbrubeck) [inline Cargo.toml]: It looks like this is a non-breaking change.  If so, this could be 0.11.2.

**Review by emilio — APPROVED** [review]

**Comment by glennw** [comment]
@emilio Is bors enabled on this repo?

**Comment by emilio** [comment]
Err, nope

**Pull request merged**